server/test/article.spec.js

```diff
@@ -174,13 +174,29 @@ describe('Articles routes', () => {
     });
     it('it should get all articles', async () => {
       const res = await chai.request(app)
-        .get('/api/v1/article')
+        .get('/api/v1/article/1')
         .set({ Authorization: `Bearer ${userToken}` });
       const { message } = res.body;
       expect(res).to.have.status(200);
       expect(message).to.be.equal('Articles successfully retrieved');
       expect(res.body.data).to.have.property('articles');
     });
+    it('it should return error if no more articles on page', async () => {
+      const res = await chai.request(app)
+        .get('/api/v1/article/4')
+        .set({ Authorization: `Bearer ${userToken}` });
```

Comment:
Good test case, but you may need to check this line, there is no need for this line because any user can view articles either authenticated or not. And in your route there is no authentication middleware used